src/transformers/generation/configuration_utils.py

```diff
@@ -281,6 +287,18 @@ class GenerationConfig(PushToHubMixin):
 
         cache_implementation (`str`, *optional*, default to `None`):
             Cache class that should be used when generating.
+        cache_config (`Union[QuantizedCacheConfig, dict]`, *optional*, default to `None`):
+            Arguments used for quantized cache that stores keys and values in lower precision for memory efficiency.
+            If passed as `Dict`, it will be converted to a `QuantizedCacheConfig` internally.
+            Accepts the following keys:
```

Comment:
TO be removed, Depending on the class it's gonna be different

Reply:
will indicate the base `CacheConfig` class